### DISCUSSION TITLE
Citation for EpiModel

### DISCUSSION THREAD

**Discussion by martinamorris** [EpiModel/EpiModel#534] [General]
For package citations, there's citation of the package itself (recommended, as it includes the version number) and citation of one or more papers that address either the use of the package (like the JoSS papers) or the methodology (e.g., for stat pkgs).

Right now, EpiModel cites just the JoSS paper (which is out of date now).

We might consider instead following the lead of the `survival` package:

```
> citation("survival")

Therneau T (2021). _A Package for Survival Analysis in R_. R package version
3.2-11, <URL: https://CRAN.R-project.org/package=survival>.

Terry M. Therneau, Patricia M. Grambsch (2000). _Modeling Survival Data:
Extending the Cox Model_. Springer, New York. ISBN 0-387-98784-3.
```

In our case, the package itself (which may only have Sam's name on it) and the Joss paper.

thoughts @smjenness @sgoodreau ?

**Comment by sgoodreau** [comment]
That makes the most sense to me.  Most R packages have both, right?

**Reply by martinamorris** [reply]
It really varies.  Utility packages like `dplyr` often just cite the package.  Statistical packages like `survival` sometimes have 2, but `flexsurv` cites a JoSS paper.  And `ggplot2` cites a book.  It feels like this is not really settled.

But for publication, following the principles of reproducibility, it seems like it would be important to cite the version of the package that was used, not a manual.

**Comment by smjenness** [comment]
I prefer people cite the JSS paper, as it is more likely to get picked up on Google scholar/iCite/WoS.
